redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -170,10 +170,14 @@ public function clearAllRecipients()
 
     private function log(string $success): void
     {
+        $replytos = '';
+        if (count($this->getReplyToAddresses()) > 0) {
+            $replytos = implode(', ', array_column($this->getReplyToAddresses(), 0));
+        }
         $log = new rex_log_file(self::logFile(), 2000000);
         $data = [
             $success,
-            $this->From,
+            $this->From.($replytos ? ', '.$replytos : ''),
```

Comment:
Vielleicht noch irgendwie kenntlich machen, dass es reply-to ist?
z.b. so:

```suggestion
            $this->From.($replytos ? '; reply-to: '.$replytos : ''),
```

Reply:
Stimmt, danke